gcp/appengine/frontend_handlers.py

```diff
@@ -515,6 +517,16 @@ def group_versions(versions):
   return groups
 
 
+def sort_version(versions):
+  """Sorts a list of version numbers in natural order."""
+  try:
+    return sorted(versions, key=Version)
```

Comment:
This only supports python versions (and ecosystems that can be coerced into python versions) right?

Reply:
Can you have a look at `osv.ecosystems` and use the version sorting logic we have there depending on the ecosystem? And for the `OrderingUnsupportedEcosystem`s just do the lexicographical sorting.

Reply:
Oh I'm a bit late haha.

Reply:
Thanks!